crates/holochain/src/test_utils/network_simulation.rs

```diff
@@ -487,58 +490,58 @@ async fn reset_peer_data(peers: Vec<AgentInfoSigned>, dna_hash: &DnaHash) -> Vec
     peer_data
 }
 
-fn get_ops(txn: &mut Transaction<'_>) -> HashMap<Arc<DhtOpHash>, DhtOpHashed> {
+fn get_chain_ops(txn: &mut Transaction<'_>) -> HashMap<Arc<DhtOpHash>, ChainOpHashed> {
     txn.prepare(
         "
                 SELECT DhtOp.hash, DhtOp.type AS dht_type,
                 Action.blob AS action_blob, Entry.blob AS entry_blob
-                FROM DHtOp
+                FROM DhtOp
                 JOIN Action ON DhtOp.action_hash = Action.hash
-                LEFT JOIN Entry ON Action.entry_hash = Entry.hash
+                JOIN Entry ON Action.entry_hash = Entry.hash
```

Comment:
This one should be a LEFT JOIN right? Because entries are optional even if this test doesn't create any ops without  entries?

Reply:
Wow, I must have been rushing, I meant to add LEFT on the line above but instead removed it on the line below!